borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,89 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
+                type: integer
+                description: |
+                    The ID of the Zabbix item used for collecting data.
+                    Unique across the entire Zabbix system.
+                example: 55105
+            host:
+                type: string
+                description: |
+                    Host name where the item is stored.
+                    Required if "itemid" is not set.
```

Comment:
Nit: These descriptions should be word-wrapped to 80 chars. (Meaning, the second sentence should begin right after the first one on the same line.)

Not a big deal though; I can always take care of these after merging when it gets to that point.

Reply:
https://github.com/borgmatic-collective/borgmatic/commit/237999cc81cb1b35dab251d4154cf9aeeb8ecc7c